test/test_helper.rb

```diff
@@ -3,11 +3,25 @@
 require 'rails/test_help'
 require 'capybara/rails'
 require 'minitest/emoji'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.